notify_send.py

```diff
@@ -160,6 +160,10 @@
         '/usr/share/icons/hicolor/32x32/apps/weechat.png',
         'Path to an icon to be shown in notifications.'
     ),
+    'desktop-entry': (
```

Comment:
All the other options use underscores (`_`) instead of dashes (`-`), so I suggest renaming `desktop-entry` to `desktop_entry`.